build/configs/rtl8730e/loadable_ext_ddr/defconfig

```diff
@@ -587,6 +587,7 @@ CONFIG_LCD_MAXPOWER=100
 #
 # CONFIG_LCD_ST7701 is not set
 CONFIG_LCD_ST7701SN=y
+CONFIG_LCD_SW_ROTATION=y
```

Comment:
I think you add this manually. When I check the Kconfig, this should be added at different line.
Could you do this using menuconfig?
Using menuconfig, you can modify .config which is in os folder
and then you can copy necessary line into the defconfig.

Reply:
fixed.